docs/databases/new.md

```diff
@@ -0,0 +1,64 @@
+---
+sidebar_position: 2
+---
+
+# Other databases
+
+If you are using a database that is not currently supported by Sequelize, there are a few options available to you:
+
+- [Request support for the new dialect](#requesting-support-for-a-new-dialect)
+- [Contribute a new dialect](#contributing-a-new-dialect)
+- [Create a third-party dialect](#creating-a-third-party-dialect)
+
+## Requesting support for a new dialect
+
+You can request support for a new dialect by creating a feature request on the [Sequelize repository](https://github.com/sequelize/sequelize).
+Make sure to verify that there isn't already an open issue for the dialect you are requesting. [View the list of requests for new dialects here](https://github.com/sequelize/sequelize/issues?q=is%3Aopen+is%3Aissue+label%3A%22dialect%3A+new%22).
+
+We have conditions for accepting new dialects:
+
+- The database must support SQL queries.
+- The database must have an existing Node.js library we can use to connect to it.
+- We must be able to run a database instance in GitHub codespaces for testing purposes.
```

Comment:
Why specifically GitHub Codespaces? I would personally go for GitHub Actions so it fits within our CI. Or do you want to explicitly mention something for local development?

Reply:
I figured it was a more objective way to determine whether we could run it locally than "can be run locally", as running locally depends on the OS, whereas everyone can access codespaces

Reply:
Done!